libs/oci/langchain_oci/chat_models/oci_generative_ai.py

```diff
@@ -397,17 +443,9 @@ def messages_to_oci_params(
         for i, message in enumerate(messages[::-1]):
             current_turn.append(message)
             if isinstance(message, HumanMessage):
-                if len(messages) > i and isinstance(
-                    messages[len(messages) - i - 2], ToolMessage
-                ):
-                    # add dummy message REPEATING the tool_result to avoid
-                    # the error about ToolMessage needing to be followed
-                    # by an AI message
-                    oci_chat_history.append(
-                        self.oci_chat_message["CHATBOT"](
-                            message=messages[len(messages) - i - 2].content
-                        )
-                    )
```

Comment:
formatting changes make it hard to see what really changed.  Please run ruff format